README.md

```diff
@@ -113,6 +113,33 @@ RUN dpkg --add-architecture arm64 && \
 $ docker build -t my/image:tag path/to/where/the/Dockerfile/resides
 ```
 
+### Passing environment variables into the build environemnt
+
+By default, `cross` does not pass any environment variables into the build
+environment from the calling shell. Sometimes this is really useful, for
+example to pass in environment variables set by your CI system.
+
+To whitelist some variables, you can specify them in your `Cross.toml`
+like so
+
+```toml
+[build.env]
+whitelist = [
+    "TRAVIS",
```

Comment:
#74 specifically talks about `RUST_BACKTRACE`. Maybe add that here as well as a hint to users that that might be useful?